config/util/helpers.go

```diff
@@ -25,3 +25,11 @@ func StrToPtr(s string) *string {
 func BoolToPtr(b bool) *bool {
 	return &b
 }
+
+func NilOrEmpty(s *string) bool {
```

Comment:
Nice! (I think those are being used in a previous commit BTW, but eh...)

Reply:
fixed